plugins/auth-backend-module-oidc-provider/src/authenticator.ts

```diff
@@ -30,6 +31,10 @@ import {
   PassportOAuthPrivateInfo,
 } from '@backstage/plugin-auth-node';
 
+custom.setHttpOptionsDefaults({
+  timeout: 10000,
+});
```

Comment:
Could we use the approach where the `custom.http_options` symbol is set on the client instead? It's kinda hard to know what other usages there might be of `openid-client` already or in the long run, so setting this globally is a bit risky.